### PR TITLE
add fie temperature.js and code to task convert celsius to fahrenheit

### PR DESCRIPTION
**Pull request by Ksenia000**

### CODE REVIEW

File: Learn/js/temperature_script.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+ function Temperatures (){ 
+ 	
+//Считаем значения из поля 
+	var data = document.getElementById("convert"). value ;
+	var degree = document.getElementById("degr"). value ;
```
Comment (akryvenko) [inline Learn/js/temperature_script.js]: remove space

File: Learn/js/temperature_script.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	var res ;
+	var dataInt ;
+	dataInt = parseInt(data, 10);
+	console.log(typeof data,data)
```
Comment (akryvenko) [inline Learn/js/temperature_script.js]: remove this

File: Learn/page2.html [hunk 3]
```diff
@@ -20,12 +21,26 @@ <h3>Write a JavaScript program to rotate the string 'w3resource' in right direct
...
+      <h3>Write a JavaScript program to determine whether a given year is a leap year in the Gregorian calendar.</h3></br>
+	  <button onClick = "fillLeapYearElement()"> Is the year leap?</button></br>
+	  <span id = "resYear"></span></br>
 	 </fieldset>
 	</div>
 	<div class = "cols col-6 colTask">
+     <fieldset>
+      <legend>Temperatures</legend>
+      <h3>Write a JavaScript program to convert temperatures to and from Celsius, Fahrenheit.</h3>
+	  <p><input type = "text" name = "tempData" id = "convert"  value = "45">
```
Comment (akryvenko) [inline Learn/page2.html]: remove redundsant spaces

**Review by akryvenko — APPROVED** [review]
Looks good

File: Learn/js/temperature_script.js (hunk 1 above)
Comment (Ksenia000) [reply Learn/js/temperature_script.js]: +

File: Learn/js/temperature_script.js (hunk 2 above)
Comment (Ksenia000) [reply Learn/js/temperature_script.js]: done

File: Learn/page2.html (hunk 3 above)
Comment (Ksenia000) [reply Learn/page2.html]: done

**Pull request merged**